### DISCUSSION TITLE
wget problem snapshot via terminal

### DISCUSSION THREAD

**Discussion by Giuserver** [roleoroleo/yi-hack-Allwinner-v2#532] [Q&A] [answered]
Hello @roleoroleo, 
I have the version 0.2.5 (last in November 2022) in my yi 1080 BFUS camera, when i get snapshots from URL in web browser:
http://192.168.43.216:8080/cgi-bin/snapshot.sh?res=high&watermark=yes **it'OK**
but from terminal when i get the command:
wget -O image.jpg -i "http://192.168.43.216:8080/cgi-bin/snapshot.sh?res=high&watermark=yes"
**start looping and the reboot itself!** 
can i resolve?

Thanks
Giuseppe
**Output example:**
############################################################
--2022-11-23 19:16:03--  http://192.168.43.216:8080/cgi-bin/(%A2%80
Connessione a 192.168.43.216:8080... connesso.
Richiesta HTTP inviata, in attesa di risposta... 404 Not Found
2022-11-23 19:16:03 ERRORE 404: Not Found.

--2022-11-23 19:16:03--  http://192.168.43.216:8080/cgi-bin/(%A2%80
Connessione a 192.168.43.216:8080... connesso.
Richiesta HTTP inviata, in attesa di risposta... 404 Not Found
2022-11-23 19:16:03 ERRORE 404: Not Found.

**Comment by roleoroleo** [comment, marked as answer]
Remove "-i" from command line.

**Reply by Giuserver** [reply]
> Remove "-i" from command line.

yes, just tried. Same problem
some time work correctly some time start looping and camera rebooting itself

**Reply by roleoroleo** [reply]
About reboot, is swap space enabled?
What happens if you call low res snapshot?

**Reply by Giuserver** [reply]
> About reboot, is swap space enabled?
> What happens if you call low res snapshot?

Perfect. 
With swap space enabled work low and high snapshots.

Thank you, again.

Giuseppe
